### PR TITLE
PA-01: Implemented Our lessons section

### PR DESCRIPTION
**Pull request by kharkov-timur**

### CODE REVIEW

**Comment by Fijji** [comment]
please git squash commits if you feel comfortable

File: src/css/base.css [hunk 1]
```diff
@@ -1,19 +1,166 @@
...
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.section {
+  padding-bottom: 80px;
```
Comment (dimafrontdev) [inline src/css/base.css:44]: should be margin

**Comment by Fijji** [comment]
please let's try to maintain consistency with file names. It would help things easier to find. Thanks 
our_lessons.html
our-lessons.css

File: src/css/our-lessons.css [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+@media screen and (max-width: 768px) {
+  .lesson-item {
+    max-width: 335px;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .container {
+    max-width: 768px;
```
Comment (mytrichq) [inline src/css/our-lessons.css]: this should be as a common rule in base.css

File: src/css/base.css [hunk 3]
```diff
@@ -1,19 +1,166 @@
...
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  margin: 0;
+  padding: 0;
+}
```
Comment (mytrichq) [inline src/css/base.css]: we already have file reset.css with such styles

File: src/css/base.css [hunk 4]
```diff
@@ -1,19 +1,166 @@
...
+a {
+  text-decoration: none;
+  color: var(--black);
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
```
Comment (mytrichq) [inline src/css/base.css]: same as above

File: src/css/base.css [hunk 5]
```diff
@@ -1,19 +1,166 @@
...
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 1024px;
+  min-width: 320px;
+  margin: 0 auto;
+}
```
Comment (mytrichq) [inline src/css/base.css:37]: as we're using mobile-first methodology, max-width should be 375px by default, then 768px for tablet and 1280px for desktop

File: src/css/base.css [hunk 6]
```diff
@@ -1,19 +1,166 @@
...
+    position: absolute;
+    display: block;
+    width: 116px;
+    height: 12px;
+    background-color: var(--lighter-green);
+    border-radius: 8px;
+    right: 0;
+    bottom: 0;
+    z-index: -1;
+  }
```
Comment (mytrichq) [inline src/css/base.css:110]: there's no need to duplicate properties that repeats inside the same rule above (display, position, display, bgcolor, radius)

File: src/css/base.css [hunk 7]
```diff
@@ -1,19 +1,166 @@
...
+
+  .section-title {
+    font-size: 40px;
+  }
+
+  .section-subtitle {
+    font-size: 18px;
+    margin-bottom: 64px;
+    max-width: 665px;
```
Comment (mytrichq) [inline src/css/base.css:137]: duplicated max-width with the same value from the rule above

File: src/css/base.css [hunk 8]
```diff
@@ -1,19 +1,166 @@
...
 
-/* .container {
-} */
\ No newline at end of file
+.section-subtitle {
+  font-size: 14px;
+  opacity: 0.5;
+  margin-bottom: 40px;
+  max-width: 665px;
+  text-align: center;
```
Comment (mytrichq) [inline src/css/base.css:66]: missing line-height

File: src/css/our-lessons.css [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 1.2;
+  margin-bottom: 16px;
+}
+
+.lesson-item-price {
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 1.2;
```
Comment (mytrichq) [inline src/css/our-lessons.css]: according to figma:
mobile - line-height: 36px; /* 112.5% */
tablet and desktop - line-height: 44px; /* 137.5% */

File: src/css/our-lessons.css [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+}
+
+.middle-item {
+  background-color: var(--green);
+  color: var(--white);
+}
+
+.middle-item .lesson-item-text {
+  color: var(--white);
+}
```
Comment (mytrichq) [inline src/css/our-lessons.css]: opacity should be 0.7 and also middle-item is not a good classname, better to name it "active-item" or something like this

File: src/css/our-lessons.css [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+  opacity: 0.5;
+  text-align: center;
+}
+
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  margin-top: auto;
```
Comment (mytrichq) [inline src/css/our-lessons.css]: not sure we need margin-top here as we center using flex

File: src/css/base.css [hunk 12]
```diff
@@ -1,19 +1,166 @@
...
+}
+
+.button {
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 1.7;
+  color: var(--white);
+  background-color: var(--green);
+  border-radius: 12px;
+  padding: 16px 32px;
```
Comment (mytrichq) [inline src/css/base.css]: for mobile vertical paddings should be 14px and for tablet/desktop 16px

**Review by mytrichq — CHANGES_REQUESTED** [review]

File: src/css/base.css (hunk 5 above)
Comment (kharkov-timur) [reply src/css/base.css:37]: This is container size not breakpoint

Comment (mytrichq) [reply src/css/base.css:37]: yes and for mobile we can't have max-width: 1024

File: src/css/our-lessons.css (hunk 11 above)
Comment (kharkov-timur) [reply src/css/our-lessons.css]: We need it for save lesson item height and fix button in the end of the item

File: src/css/base.css (hunk 12 above)
Comment (kharkov-timur) [reply src/css/base.css]: Done

File: src/css/our-lessons.css (hunk 10 above)
Comment (kharkov-timur) [reply src/css/our-lessons.css]: Done

File: src/css/our-lessons.css (hunk 9 above)
Comment (kharkov-timur) [reply src/css/our-lessons.css]: Done

File: src/css/base.css (hunk 8 above)
Comment (kharkov-timur) [reply src/css/base.css:66]: Done

File: src/css/base.css (hunk 4 above)
Comment (kharkov-timur) [reply src/css/base.css]: Done

File: src/css/base.css (hunk 3 above)
Comment (kharkov-timur) [reply src/css/base.css]: Done

File: src/css/our-lessons.css (hunk 2 above)
Comment (kharkov-timur) [reply src/css/our-lessons.css]: Done

File: src/css/our-lessons.css [hunk 13]
```diff
@@ -0,0 +1,106 @@
...
+    opacity: 0.5;
+    text-align: center;
+  }
+}
+
+@media screen and (min-width: 1280px) {
+  .container {
+    max-width: 1024px;
+    padding: 0;
+  }
```
Comment (mytrichq) [inline src/css/our-lessons.css]: move it to base.css

File: src/css/our-lessons.css [hunk 14]
```diff
@@ -0,0 +1,106 @@
...
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  margin-top: auto;
+}
+
+.lesson-item-button .button {
+  width: 100%;
```
Comment (mytrichq) [inline src/css/our-lessons.css]: there's no such class in html

File: src/css/base.css (hunk 5 above)
Comment (kharkov-timur) [reply src/css/base.css:37]: Done

File: src/css/base.css (hunk 6 above)
Comment (kharkov-timur) [reply src/css/base.css:110]: Done

File: src/css/our-lessons.css (hunk 13 above)
Comment (kharkov-timur) [reply src/css/our-lessons.css]: Done

File: src/css/our-lessons.css (hunk 14 above)
Comment (kharkov-timur) [reply src/css/our-lessons.css]: Done

**Review by mytrichq — APPROVED** [review]

**Pull request merged**